tasks/main.yml

```diff
@@ -1,9 +1,22 @@
 ---
+#Probe pv partitions to detect size changes
+- name: "Probe partitions"
+  parted:
```

Comment:
If we're being explicit, perhaps `community.general.parted` ?

Reply:
Good point!